human-eval-benchmark/Readme.md

```diff
@@ -0,0 +1,27 @@
+# How to Run the LLM benchmark - Inference workload
+
+There are two ways to run the LLM benchmark:
+
+1. Workbench - Jupyter Notebook
+2. Automated Job
+
+## 1. Using the Workbench 
+## Pre-requisite 
+As of now the user needs to have access to openshift AI cluster, create a Data Science project and setup a workbench with the following configuration 
```

Comment:
Can you elaborate on openshift AI cluster. For example, an OCP cluster with GPU nodes and 'xyz' operator installed